packages/fx-core/src/plugins/solution/fx-solution/constants.ts

```diff
@@ -295,3 +295,24 @@ export const TEAMS_APP_SHORT_NAME_MAX_LENGTH = 30;
 export const DEFAULT_DEVELOPER_WEBSITE_URL = "https://www.example.com";
 export const DEFAULT_DEVELOPER_TERM_OF_USE_URL = "https://www.example.com/termofuse";
 export const DEFAULT_DEVELOPER_PRIVACY_URL = "https://www.example.com/privacy";
+
+export enum SolutionTelemetryEvent {
+  CreateStart = "create-start",
+  Create = "create",
+
+  AddResourceStart = "add-resource-start",
+  AddResource = "add-resource",
+
+  AddCapabilityStart = "add-capability-start",
+  AddCapability = "add-capability",
+}
+
+export enum SolutionTelemetryProperty {
+  Component = "component",
+  Resources = "resources",
+  ResourcesToAdd = "resources-to-add",
```

Comment:
feel no need for "ResourceToAdd" and "CapabilitiesToAdd", just use "Resources" and "Capabilities". People should know the meaning.

Reply:
"Resources" alone could be ambiguous, because it could mean "resources that have already been added" and "resources to add"

Reply:
that's why you need "success": "yes/no". I believe for "yes" means resources been added. 'no' means to be added.